fuzzing/CMakeLists.txt

```diff
@@ -1,38 +1,59 @@
-cmake_minimum_required(VERSION 3.10)
-
-project(Fuzzer
-        VERSION 1.0
-	      DESCRIPTION "SDK Fuzzer"
-        LANGUAGES C)
+cmake_minimum_required(VERSION 3.14)
 
 set(CMAKE_C_COMPILER clang)
```

Comment:
Do we need those lines? The README already set clang as an argument.
@bboilot-ledger what would be easier for devs?